### DISCUSSION TITLE
Adding an anchor to Pagerfanta Twig options ?

### DISCUSSION THREAD

**Discussion by bricebou** [BabDev/Pagerfanta#46] [Q&A] [answered]
Hi,

I'm working on a small Symfony project as a self-taught exercise and I'm trying to figure out how to proceed to link to an anchor when clicking on a pagerfanta pagination link.

I've found this https://github.com/whiteoctober/Pagerfanta/issues/108 but it's quite old ; maybe there is an option I haven't found ?

**Comment by mbabker** [comment, marked as answer]
All the variables available in the Twig templates are documented at https://www.babdev.com/open-source/packages/pagerfantabundle/docs/3.x/views#creating-a-twig-view-template (I really should move that stuff from the bundle to the library).

Your best bet is going to be to extend the appropriate Twig template and replace the `*_page_link` blocks with a version that includes your anchor (how the anchor gets generated is going to be on you to sort out).

**Reply by bricebou** [reply]
Thanks a lot for your answer and for Pagerfanta :)
